docs/manuals/gui/corelements/data_node_selector.md_template

```diff
@@ -13,6 +13,51 @@ scenario level, here is what the data node selector would look like:
   <figcaption>The list of selectable data nodes</figcaption>
 </figure>
 
-When the user selects a data node, the [*on_change*](#p-on_change) callback is invoked, so that
+Data nodes are organized in their owning scenario and cycle, when relevant.
+
+When the user selects a data node, the [*on_change*](#p-on_change) callback is invoked so that
 the application can use the selected value. The value is set to the [*value*](#p-value) property.
 
+<h2>Pins</h2/>
+
+When there are many data nodes in your application, the user can filter out a set of data nodes
+by *pinning* them and then set the *Pinned only* switch (that is active only if some data nodes are
+pinned): only pinned data nodes will then appear in the list.
+
+Assuming we are in the following situation:
+<figure class="tp-center">
+  <img src="../data_node_selector-pin1-d.png" class="visible-dark"  />
+  <img src="../data_node_selector-pin2-l.png" class="visible-light" />
+  <figcaption>Crowded data node selector</figcaption>
+</figure>
+
+If the user wants to focus only on the 'initial_dataset' and the data nodes from the scenario
+called 'Peter's', she can click on the pin icon next to these two items. Here is what the display
+would look like:
+<figure class="tp-center">
+  <img src="../data_node_selector-pin2-d.png" class="visible-dark"  />
```

Comment:
We have a redundant picture here. `data_node_selector-pin2-d.png` is the same as the one before `data_node_selector-pin1-d.png`. Is it on purpose?
I would change `data_node_selector-pin1-d.png` so No data node is pinned.

Reply:
Actually these pictures ARE different.
pin1 has no pinned DNs.
pin2 has 'initial_dataset' and 'Peter's' scenario

Reply:
The problem was in the reference to the pictures. Thanks!